prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -63,7 +63,10 @@ public async Task<ActionResult> GetEnrollees([FromQuery] EnrolleeSearchOptions s
         {
             if (User.HasAdminView())
             {
-                return Ok(ApiResponse.Result(await _enrolleeService.GetEnrolleesAsync(searchOptions)));
+                var notifiedIds = await _enrolleeService.GetNotifiedEnrolleeIdsForAdminAsync(User);
+                var enrollees = await _enrolleeService.GetEnrolleesAsync(searchOptions);
+                var result = enrollees.Select(e => notifiedIds.Contains(e.Id) ? e.SetNotification(true) : e.SetNotification(false));
```

Comment:
Here you could just do
`foreach(var enrollee in enrollees)
{
    e.HasNotification = notifiedIds.Contains(enrollee.Id)
}
`